DuckDuckGo/BookmarksViewController.swift

```diff
@@ -191,9 +191,7 @@ class BookmarksViewController: UITableViewController {
     fileprivate func showShareSheet(for indexPath: IndexPath) {
 
         if let link = currentDataSource.link(at: indexPath) {
-            let appUrls: AppUrls = AppUrls()
-            let url = appUrls.removeATBAndSource(fromUrl: link.url)
-            presentShareSheet(withItems: [ url, link ], fromView: self.view)
+            presentShareSheet(withItems: [link], fromView: self.view)
```

Comment:
The `Link` instance is calling `removeATBAndSource` itself, so we no longer need to do that here. 

https://github.com/duckduckgo/iOS/blob/develop/Core/UIViewControllerExtension.swift#L86

Reply:
Yes, that's right, but testing that made me realize we have another bug :) 

For sharing search URLs, can you make it so `public func removeATBAndSource(fromUrl url: URL) -> URL`  also removes `Param.searchHeader` (ko) otherwise search bar won't show in other browsers.

Reply:
@bwaresiak Great find, fixed and updated tests. I renamed `removeATBAndSource` to `removeInternalSearchParameters`, happy to change this to something else though. 🙂